modules/extensions-support/src/main/java/org/mule/module/extension/internal/manager/DefaultExtensionManager.java

```diff
@@ -262,6 +294,41 @@ private void logExtensionHotUpdate(Extension extension, Extension actual)
         }
     }
 
+    private ConfigurationExpirationMonitor newConfigurationExpirationManager()
```

Comment:
can you inject the ConfigurationExpirationMonitor instead of having this dependency?